core/tests/protractor_utils/LibraryPage.js

```diff
@@ -99,9 +99,9 @@ var LibraryPage = function() {
     await browser.actions().mouseMove(element(by.css(
       '.protractor-test-exp-summary-tile-title'))).perform();
```

Comment:
Can you wait for the element here to be visible before we interact with it like this?